packages/host/app/services/loader-service.ts

```diff
@@ -47,70 +52,40 @@ export default class LoaderService extends Service {
       new URL(baseRealm.url),
       new URL(config.resolvedBaseRealmURL),
     );
-    loader.prependURLHandlers([(req) => this.fetchWithAuth(req)]);
+    loader.prependURLHandlers([
+      addAuthorizationHeader({
+        getLoader: () => {
```

Comment:
Can you just pass the loader directly as an arg? Are you expecting loaders to change?